### PR TITLE
This closes  #16, closes #14

### LINKED ISSUE #14: Make new interface responsive for different screen sizes
- Need to add few media queries to make the site responsive and usable for every screensize
PR's are welcomed!

### LINKED ISSUE #16: Structure to organise the static assets
Currently there is not file structure. Why don't we structure it to something like this !

```
static/
         images/
         scripts/ # for js files
         styles/ # for css files
  
```

### PR DESCRIPTION
**Pull request by rrmerugu**
Demo  : http://fcc-hyd-v001.surge.sh/

### CODE REVIEW

**Comment by chakrihacker** [comment]
I don't think there is a need for devdocs

**Comment by duttakapil** [comment]
Thanks for making this PR so quickly @rrmerugu. Can you also host this on surgh.sh and paste the demo link in your description? Thanks

**Comment by rrmerugu** [comment]
@vramana  css prettified.

**Comment by rrmerugu** [comment]
@duttakapil  I will do that, let me explore surge.sh, i will do that in some time.

**Comment by rrmerugu** [comment]
Here is the demo link fcc-hyd-v001.surge.sh

**Comment by rrmerugu** [comment]
@chakrihacker I wrote the dev doc,  with only one intention. To make sure everyone understands the convention I followed in the code. When a code is implemented by multiple coders, especially in opensource contributions like this. There is a chance of multiple conventions being used, and people getting lost and irritated with multiple conventions. So I conveyed my style of implementation. When we finalise a standard practise of writing code for this project, then I still see the need for a developer doc or style doc for that finalised style.

**Comment by vramana** [comment]
@rrmerugu You can include any contributing tips in CONTRIBUTING.md. That's the github convention.

**Comment by vramana** [comment]
Also It's better to use CDN and not commit the Minified files to the repo. Upgrading version becomes a breeze. If we are using a CDN like unpkg it will resolve to the latest version of the library. So we will always staty up to date.

File: index.html [hunk 1]
```diff
@@ -4,80 +4,191 @@
...
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="./"><img src="./static/images/logo.png" height="80"></a>
+    </div>
+
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
```
Comment (vinaypuppal) [inline index.html]: Can you give a better `id` instead of `bs-example-navbar-collapse-1`

File: index.html [hunk 2]
```diff
@@ -4,80 +4,191 @@
...
-      <h1>Facebook Developer Circle Launch</h1>
+</section>
+
+
+<section class="">
+  <div class="container">
+    <div class="row ">
+      <div class="col-sm-8">
+
+        <div class="upcoming-meetup" style=" ">
```
Comment (vinaypuppal) [inline index.html]: Remove all unused `style` attributes

**Comment by vinaypuppal** [comment]
I think site is still not responsive approx form `1200px` to `780px` screen size few parts looks like these
- screen shot 2017-03-15 at 9 48 09 pm

- screen shot 2017-03-15 at 9 48 25 pm

**Comment by rrmerugu** [comment]
@vinaypuppal updated the responsiveness http://fcc-hyd-v001.surge.sh .

**Comment by vramana** [comment]
@rrmerugu Please respond to my comments on using a CDN instead of commiting files.

**Comment by rrmerugu** [comment]
@vramana CDN is a great idea, we can do that. But I'm not really inclined to using the latest source for every framework. Because sometime(not always) the dependencies might be broken because of using the latest. But I see that we can use version specific CDN ? That way, we will not have any surprise when some of the frameworks we are using upgrades their version to latest. 

This happened to me in one of the projects I was using with AngularJS. I was using angular latest and when angular updated to 1.5, few components broke in the production and that was a nightmare. So I love CDNifying. But I believe we have control over what very we use, instead of calling the latest CDN. 

Comments appreciated.  


**Also** I have seen a discussion over why use bootstrap. I see people are seeing bootstrap as a huge framework, but they give you flexibilty to customize it. We can customise the bootstrap using bootstrap customize tool with only few components we need. This we wil not CDN, so we might have to use the code itself like I did now.

File: static/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
+*,
```
Comment (vramana) [inline static/styles/style.css]: Isn't this already part of bootstrap?

**Comment by vramana** [comment]
@rrmerugu Your reasoning doesn't make sense to me. jQuery and bootstrap are pretty stable in my opinion and     We can lock down the major version. That should prevent any breaking changes. 

I have never said size was the issue. Even before knowing what the site would look like why discuss about the using tailored version of bootstrap. If we do that now it will be continuous maintanence burden. 

I have to be honest that even mine feels flawed as well. During development all this feels unnecessary to discuss anyway. Just push through features rather than bikeshedding infinitely. Once there is final product, we can revisit these.

**Comment by duttakapil** [comment]
@rrmerugu if you compare [your demo](http://fcc-hyd-v001.surge.sh/) with the [current interface](https://fcc-hyd.github.io/fcc-hyd/), you can notice quite some differences. Mainly with the spacing between different sections, the navbar interface, and the logo size. We definitely need to control the spacing on smaller screens. 

Can you put your dev-notes.md content in CONTRIBUTING.md file? We'll have to create one anyway. We can include things like using surge for demo, creating a new branch if you're working on a issue etc as well.

**Comment by rrmerugu** [comment]
@vramana  I understand, for the current state of the product, my reasoning might not be important. But I always prefer a strict version based approach instead of using  **latest** versions. Since I come python background, Im so into freezing the version instead of the latest. So I used some CDN  now or should we leave this as is ? Even if we start which version of Jquery to use,  it will become a very big discussion :(  let me know

**Comment by rrmerugu** [comment]
@duttakapil  i will update by tonight

**Comment by vramana** [comment]
@rrmerugu You can leave as is. Keep the good work going.

File: static/styles/style.css (hunk 3 above)
Comment (rrmerugu) [inline static/styles/style.css]: not sure. I just left it from previous code.

**Comment by rrmerugu** [comment]
@duttakapil 

- i moved the dev notes to CONTRIBUTING.md
- I'm not sure if I can handle that styling for next 2 days, my bandwidth is too low.

**Comment by duttakapil** [comment]
Ok. So should we merge this, then work on backtracking to current UI?
We can create a issue specifically for that and share in the group, after merge.

**Comment by rrmerugu** [comment]
@duttakapil Yeah, I think its a good to merge and handle the spacing/UI as an issue.

**Pull request merged**